src/hotspot/share/gc/shared/gcVMOperations.hpp

```diff
@@ -37,19 +37,22 @@
 // a set of operations (VM_Operation) related to GC.
 //
 //  VM_Operation
-//    VM_GC_Sync_Operation
+//    VM_Heap_Sync_Operation
 //      VM_GC_Operation
-//        VM_GC_HeapInspection
-//        VM_PopulateDynamicDumpSharedSpace
-//        VM_SerialGCCollect
-//        VM_ParallelGCCollect
-//        VM_CollectForAllocation
-//          VM_SerialCollectForAllocation
-//          VM_ParallelCollectForAllocation
+//        VM_GC_Collect_Operation
+//          VM_SerialGCCollect
+//          VM_ParallelGCCollect
+//          VM_CollectForAllocation
+//            VM_SerialCollectForAllocation
+//            VM_ParallelCollectForAllocation
```

Comment:
```suggestion
//          e.g. VM_SerialGCCollector
//          VM_CollectForAllocation
//            e.g. VM_SerialCollectForAllocation
```

This is not a real suggestion, but this hierarchy isn't complete, and I am not sure we should make it complete. Since we already skip Z/G1/Shenandoah operations here, I do not really see that Serial/Parallel specific operations should be here either.

Or maybe explicitly state Serial/Parallel vm ops as examples here as suggested above, but that does not look right either.

However this is kind of unrelated to this PR :)